.github/workflows/ci.yml

```diff
@@ -3,25 +3,66 @@
 
 name: continuous-integration
 
-on: [push, pull_request]
+on:
+    push:
+        branches:
+            - master
+    pull_request:
+
+env:
+    FORCE_COLOR: '1'
+
+# https://docs.github.com/en/actions/using-jobs/using-concurrency
+concurrency:
+    # only cancel in-progress jobs or runs for the current workflow - matches against branch & tags
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: true
 
 jobs:
 
-    pre-commit:
```

Comment:
The pre-commit job is not needed since we run it through `pre-commit.ci` app anyway. (we've removed this job in all other repos as well)

Reply:
makes sense 👍